src/framework/services/filter/FilterOption.js

```diff
@@ -6,6 +6,8 @@ export default class FilterOption {
     this.comparisonType = config.comparisonType;
     this.getValue = config.getValue;
     this.name = config.name;
+    // eslint-disable-next-line
+    this.isRemovable = config.isRemovable === false ? false : true;
```

Comment:
This line can be replaced with `this.isRemovable = config.isRemovable !== false;`, but then it will be less clear. I'm fine with how it is now.